source/develop/release-management/features/network/openstack_identity_api_v3.html.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Support OpenStack Identity API v3
+category: feature
+authors: dholler
+feature_name:  Support OpenStack Identity API v3
+feature_modules: engine,provider-ovn
+feature_status: In Development
+---
+
+# Support OpenStack Identity API v3
+
+## Summary
+
+[OpenStack Identity API v2.0][8] is not only deprecated anymore, it is even
+removed from the current [OpenStack Identity API specification][1] and replaced
+by the OpenStack Identity API v3.
+
+oVirt Engine requires the OpenStack Identity API to access external logical
+networks provided [OpenStack Neutron][2] or [OVN][3], to access images from
+[OpenStack Glance][4] and volumes from [OpenStack Cinder][5].
+
+The [oVirt OVN Provider][6] provides the OpenStack Identity API for
+[authentication][7].
+
+Support for OpenStack Identity API v3 in oVirt Engine enables Engine to access
```

Comment:
we do not support v3 yet, so maybe:
enables -> will enable

Reply:
done